src/YieldDistributor.sol

```diff
@@ -108,12 +109,13 @@ contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
      * @return uint256 The voting power of the user
      */
     function getCurrentVotingPower(address _account) public view returns (uint256) {
-        return this.getVotingPowerForPeriod(
-            BREAD, lastClaimedBlockNumber - cycleLength, lastClaimedBlockNumber, _account
-        )
-            + this.getVotingPowerForPeriod(
-                BUTTERED_BREAD, lastClaimedBlockNumber - cycleLength, lastClaimedBlockNumber, _account
-            );
+        uint256 lastCycleStart = lastClaimedBlockNumber - cycleLength;
+        uint256 multiplier = this.getTotalMultipliers(_account);
+        multiplier = multiplier == 0 ? 1e18 : multiplier;
+        uint256 breadVotingPower = this.getVotingPowerForPeriod(BREAD, lastCycleStart, lastClaimedBlockNumber, _account);
+        uint256 butteredBreadVotingPower =
+            this.getVotingPowerForPeriod(BUTTERED_BREAD, lastCycleStart, lastClaimedBlockNumber, _account);
+        return (breadVotingPower + butteredBreadVotingPower) * multiplier / PRECISION;
```

Comment:
Ok I see what's going on. TBH I don't think `PRECISION` should be shared here since it's being used in two different calculations. I think the `getTotalMultipliers` function should probably return a tuple like `(multiplier, multiplier_precision)` and that should be used here. Ignore my previous comment about using "1 instead of 1e18"

Reply:
so you're saying each multiplier should have it's own precision? potentially?

Reply:
I think we should return a single number formatted to 1e18, to avoid confusion and enforce standardization

Reply:
I think we all agree on this, resolving convo